packages/core/src/components/document-card/document-card.scss

```diff
@@ -0,0 +1,92 @@
+@use "~dso-toolkit/src/utilities";
+@use "~dso-toolkit/src/variables/media-query-breakpoints";
+@use "~dso-toolkit/src/variables/units";
+@use "~dso-toolkit/src/components/document-card";
+
+@include utilities.box-sizing();
+
+:host {
+  display: block;
+}
+
+.dso-document-card-container {
+  padding-block: units.$u2 units.$u1;
+  padding-inline: units.$u2;
+  inline-size: 100%;
+
+  .dso-document-card-heading {
+    align-items: center;
+    display: flex;
+
+    @media screen and (max-width: media-query-breakpoints.$screen-xs-min) {
+      flex-wrap: wrap;
+    }
+  }
+
+  &:has(.dso-document-card-heading a:is(:hover, :focus-visible)) {
+    background-color: document-card.$background-color-hover;
+  }
+}
+
+:host([active]:not([active="false"])) {
+  .dso-document-card-container {
+    background-color: document-card.$background-color-active;
+  }
+}
+
+::slotted([slot="meta"]) {
+  display: flex !important;
+  justify-content: space-between !important;
+  margin-inline-start: auto !important;
+
+  @media screen and (max-width: media-query-breakpoints.$screen-xs-min) {
+    flex-basis: 100% !important;
+    margin-block-start: units.$u2 !important;
+  }
+}
+
+::slotted([slot="heading"]) {
+  align-items: center !important;
+  display: flex !important;
+  font-size: 1.125em !important;
+  margin-block-end: 0 !important;
+  margin-block-start: 0 !important;
+}
+
+.dso-document-card-heading a {
+  text-decoration: none;
+
+  &:hover {
+    color: document-card.$heading-anchor-hover-color;
+    text-decoration: underline;
+  }
+}
+
+.heading-anchor {
+  color: document-card.$heading-anchor-color;
+  display: flex;
+  gap: units.$u1 * 0.5;
+
+  &:visited {
+    color: document-card.$heading-anchor-color;
+  }
+
+  dso-icon {
+    flex-shrink: 0;
+  }
+
+  .sr-only {
+    @include utilities.sr-only();
+  }
+}
+
+::slotted([slot="type"]) {
+  display: flex !important;
+  gap: units.$u1 !important;
```

Comment:
deze gap zit nu een beetje in de weg. Ik had de `display: flex` (zonder gap) geintroduceerd om de toggletip op de gewenste afstand van de tekst te krijgen, maar nu staat-ie er weer 8px teveel vanaf: 
<img width="444" alt="Scherm­afbeelding 2024-10-11 om 11 17 06" src="https://github.com/user-attachments/assets/3f596584-c3af-419e-a516-baf890148f6a">
Aangezien de toggletip een ingebouwde afstand heeft, kan ik 'm er nu niet meer dichterbij krijgen zonder weer te weinig afstand tussen de 2 tekstvelden te veroorzaken. breinkrakertje.

Reply:
Aha, toggletip duwt zelf ook...

He bah